crates/codegen/parser/runtime/src/kinds.rs

```diff
@@ -13,11 +13,17 @@ use napi_derive::napi;
     strum_macros::Display,
     strum_macros::EnumString,
 )]
+#[strum(use_phf)]
 #[cfg_attr( feature = "slang_napi_interfaces", /* derives `Clone` and `Copy` */ napi(string_enum, namespace = "kinds") )]
 #[cfg_attr(not(feature = "slang_napi_interfaces"), derive(Clone, Copy))]
 pub enum TokenKind {
     SKIPPED,
+    Identifier,
     // Expanded by the template engine
```

Comment:
nit: I suggest moving/rewording the older comment, as it now precedes the wrong node. Maybe moving it to the `TokenKind` enum type doc comments?

Reply:
Done